web/modules/components/src/Sprite.svelte

```diff
@@ -23,6 +29,13 @@
   }
 </script>
 
-<svg class="wj-ui-sprite sprite-{i}" {viewBox} {...$$restProps} aria-hidden="true">
+<svg
+  class="wj-ui-sprite sprite-{i}"
+  width={size}
+  style="margin: {margin};"
+  {viewBox}
+  {...$$restProps}
+  aria-hidden="true"
```

Comment:
Why is this after `restProps`?

Reply:
lol idk man this stuff happens

It doesn't actually matter, but at some point I'll probably go through a cleaning/documentation pass.